File: app/cs50x2024/content/spanish/lectures_source_code/src6/1/conditions.c
// Condiciones y operadores relacionales

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Solicitar al usuario x
    int x = get_int("x: ");

    // Solicitar al usuario y
    int y = get_int("y: ");

    // Comparar x e y
    if (x < y)
    {
        printf("x es menor que y\n");
    }
    else if (x > y)
    {
        printf("x es mayor que y\n");
    }
    else
    {
        printf("x es igual a y\n");
    }
}

File: app/cs50x2024/content/spanish/lectures_source_code/src6/1/conditions.py
# Condiciones y operadores relacionales

from cs50 import get_int

# Solicita al usuario un valor para x
x = get_int("x: ")

# Solicita al usuario un valor para y
y = get_int("y: ")

# Compara x e y
if x < y:
    print("x es menor que y")
elif x > y:
    print("x es mayor que y")
else:
    print("x es igual a y")